examples/basic_bot_functions.py

```diff
@@ -0,0 +1,104 @@
+from pyboy.utils import WindowEvent
+import pyboy as pyboy
+import io
+
+'''
+This shows the basic functions you can use if you want to write a bot using only the game image as a human would see it.
+Uses Super Mario Land 1 as an example.
+'''
+
+path_to_rom = '../SuperMarioLand1.gb'
+
+
+def save_state_to_file(boy, path):
+	boy.save_state(open(path, "wb"))
+
+
+def load_state_from_file(boy, path, reset_keys=False):
+	boy.load_state(open(path, "rb"))
+	if reset_keys:
+		for key in range(WindowEvent.RELEASE_ARROW_UP, WindowEvent.RELEASE_BUTTON_START + 1):
+			boy.send_input(key)
+
+
+def save_state(boy):
+	file_like_object = io.BytesIO()
+	file_like_object.seek(0)
+	boy.save_state(file_like_object)
```

Comment:
Did you mean to flip the order of `seek` and `save`?

Reply:
I was unsure about that as well. The only thing I found in your documentation was the following:
# Save to memory
import io
file_like_object = io.BytesIO()
file_like_object.seek(0)

so I just copied that. That's why I created this file, because I could not find any examples of how to use these functions properly.